### PR TITLE
chore: added debug logs

### PR DESCRIPTION
**Pull request by nkoppisetty**
Signed-off-by: Nandita Koppisetty <nandita.iitkgp@gmail.com>

Explain what this PR does.

### CODE REVIEW

File: numaprom/prometheus.py [hunk 1]
```diff
@@ -92,7 +93,15 @@ def query_range_limit(
...
+        try:
+            response = requests.get(
+                self.PROMETHEUS_SERVER + "/api/v1/query", params={"query": query}
+            )
+            if response:
+                results = response.json()["data"]["result"]
+            else:
+                LOGGER.debug("Prometheus query has returned empty results.")
+        except Exception as ex:
+            LOGGER.exception("error: %r", ex)
```
Comment (ab93) [inline numaprom/prometheus.py:105]:
```suggestion
            LOGGER.exception("Prometheus query error: %r", ex)
```

File: numaprom/udf/window.py [hunk 2]
```diff
@@ -12,7 +13,7 @@
 from numaprom.redis import get_redis_client
 from numaprom.tools import msg_forward, create_composite_keys, get_metric_config
 
-_LOGGER = logging.getLogger(__name__)
+LOGGER = logging.getLogger(__name__)
```
Comment (ab93) [inline numaprom/udf/window.py:16]: any reason for this? Having an underscore as a prefix basically means that this variable is internal to the file itself and should not be imported

**Review by ab93 — COMMENTED** [review]
Minor comments

File: numaprom/udf/window.py (hunk 2 above)
Comment (nkoppisetty) [reply numaprom/udf/window.py:16]: aah okay, i just changed to make it same everywhere. Will make it _LOGGER on all the files.

**Review by ab93 — APPROVED** [review]

**Pull request merged**